CRM/Civixero/Invoice.php

```diff
@@ -48,10 +48,16 @@ public function pull($params) {
           // The return syntax puts the contact only level higher up when only one contact is involved.
           $invoices = array($invoices);
         }
+        $prefix = $this->getSetting('xero_invoice_number_prefix');
+        if(!isset($prefix)) {
```

Comment:
$prefix will always be set by the preceding line so this if should not be needed